docs/changelog.md

```diff
@@ -8,6 +8,7 @@ Features
 
 - Restored the `output` attribute of &lt;py-script&gt; tags to route `sys.stdout` to a DOM element with the given ID. ([#1063](https://github.com/pyscript/pyscript/pull/1063))
 - Added a `stderr` attribute of &lt;py-script&gt; tags to route `sys.stderr` to a DOM element with the given ID. ([#1063](https://github.com/pyscript/pyscript/pull/1063))
+- Added a `docked` field and attribute for the `<py-terminal>` custom element, enabled by default when the terminal is in `auto` mode, and able to dock the terminal at the bottom of the page with auto scroll on new code execution.
```

Comment:
We will need to add a new section for the next release and add this there, I'm happy to do it as a follow up 👍

Reply:
Sure, I wasn't sure which release should I mention as I don't know *when* next release is due ... I can write it down already to not confuse anyone in the future.

Reply:
Moved!